### ISSUE TITLE
Generated Setup.exe throws InvalidOperationException as the current thread has no dispatcher associated with it.

### ISSUE THREAD

**Issue by Folcon** [Squirrel/OldSquirrelForWindows#45]
Is there some part of the process I'm missing, I've managed to build the setup.exe, however when I run it crashes with a `InvalidOperationException` as the current thread has no dispatcher associated with it.
Trace appears below:

```
at System.Reactive.Concurrency.DispatcherScheduler.get_Current()
at Shimmer.WiXUi.App.Run()
at System.Threading.ThreadHelper.ThreadStart_Context(Object state)
at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state, Boolean preserveSyncCtx)
at System.Threading.ExecutionContext.Run(ExecutionContext executionContext, ContextCallback callback, Object state)
at System.Threading.ThreadHelper.ThreadStart()
```

**Comment by shiftkey** [comment]
This will be resolved once I publish v0.6.3 to NuGet!

**Comment by anaisbetts** [comment]
@Folcon Shimmer isn't finished yet, this is one of the last pieces that needs work, getting Setup.exe to be reliable.

**Comment by shiftkey** [comment]
@xpaulbettsx this issue specifically was something I found and fixed here ebbb1f07f34d552ba5334ad61bb6551bd1ee0f84

Calling `app.Run` before the dispatcher was set was causing Bad Things.
